src/features/pages/options/digital/digits/_faq-schema.ts

```diff
@@ -0,0 +1,90 @@
+import { localize } from 'components/localization'
+import { FaqSchemaType } from 'features/types/faq-schema'
+
+export const faq_schema: FaqSchemaType = {
+    '@context': 'https://schema.org',
+    '@type': 'FAQPage',
+    mainEntity: [
+        {
+            '@type': 'Question',
+            name: localize(
+                '_t_Which Deriv trading platforms can I use to trade Digits contracts?_t_',
+            ),
+            acceptedAnswer: {
+                '@type': 'Answer',
+                text: localize(
+                    '_t_You can trade Digits contracts on the Deriv Trader, SmartTrader, and Deriv Bot trading platforms, on synthetics._t_',
+                ),
+            },
+        },
+        {
+            '@type': 'Question',
+            name: localize('_t_What are the steps to open a Digits trade on Deriv?_t_'),
+            acceptedAnswer: {
+                '@type': 'Answer',
+                text: localize(
+                    '_t_<ul><li>1. Select the market and asset you want to trade. </li><li>2. Choose which Digits contract to open (Matches/Differs; Even/Odd; Over/Under). </li><li>3. Set your preferred tick duration and last digit prediction. </li><li>4. Enter your stake or desired potential payout amount. </li><li>5. Open your Digits trade.</li></ul>_t_',
+                ),
+            },
+        },
+        {
+            '@type': 'Question',
+            name: localize('_t_Can I close a Digits contract before the expiry time?_t_'),
+            acceptedAnswer: {
+                '@type': 'Answer',
+                text: localize(
+                    '_t_No, the trade types allow you to choose the duration or expiry time but the contract will be held till the end of duration or expiry selected._t_',
+                ),
+            },
+        },
+        {
+            '@type': 'Question',
+            name: localize('_t_What are Digits contracts?_t_'),
+            acceptedAnswer: {
+                '@type': 'Answer',
+                text: localize(
+                    `_t_<p>With a Digits contract, you predict whether the last digit of the last tick of your contract will meet specified criteria, depending on the contract type you select.</p>
+                    <p>If you trade a Matches/Differs contract, you predict whether the last digit will match or be different from a selected number.</p>
+                    <p>If you trade an Even/Odd contract, you predict whether the last digit will be an even or odd number.</p>
+                    <p>If you trade an Over/Under contract, you predict whether the last digit will be over or under a specific number.</p>_t_`,
```

Comment:
`(backtick) strings don't work for translation. I had this issue earlier. As I checked, these strings are not in crowdin/messages.json
```suggestion
                    "_t_<p>With a Digits contract, you predict whether the last digit of the last tick of your contract will meet specified criteria, depending on the contract type you select.</p><p>If you trade a Matches/Differs contract, you predict whether the last digit will match or be different from a selected number.</p><p>If you trade an Even/Odd contract, you predict whether the last digit will be an even or odd number.</p><p>If you trade an Over/Under contract, you predict whether the last digit will be over or under a specific number.</p>_t_`,
```